app/views/posts/show.html.erb

```diff
@@ -1 +1,36 @@
-<h1>Displays selected post by an given user</h1>
\ No newline at end of file
+<ul class="users_container">
+  <li class="user_all_posts">
+    <div class="user_bio">
+      <div class="post_header">
+        <h3>Posted by <%= @user.Name%></h3>
+        <div class="user_recent_posts">
+          <span>
+            Comments: <%= @post.comments.count %>,
+          </span>
+          <span>
+            Likes: <%= @post.likes.count %>
+          </span>
+        </div>
+      </div>
+      <p><%= @post.Text %></p>
+    </div>
+
+    <div class="user_post_comments">
+      <% if @post.comments.empty? %>
+        <div class="user_post_comment">
+          <p>No comments</p>
+        </div>
+      <% else %>
+        <% @post.comments.each_with_index do |comment, index| %>
+          <div class="user_post_comment">
+            <h3><%= index + 1 %>. <%= comment.user.Name %></h3>
+            <p><%= comment.Text %></p>
+          </div>
+        <% end %>
+      <% end %>
```

Comment:
- [ ] Great job on your implementation so far! 🚀 However, there are a few minor issues that need some attention to ensure your code functions as expected. 

   - It appears that the data from the database isn't displaying correctly in the UI. This issue arises from the way you're referencing your data column names. I've noticed that you're using capital letters at the beginning of the column names, but they should match how they are saved in the database, typically with lowercase letters. 

   For example, in the highlighted code, you can access the `name` from your database like this:

   ```ruby
       <h3>Posted by <%= @user.name%></h3>
   ``` 

   Instead of:

   ```ruby
        <h3>Posted by <%= @user.Name%></h3>
   ``` 

   - Kindly make sure to make this correction across your entire codebase. Some of the column names you need to correct in this file include: `name`, and`text`.

Happy coding! 😊👨‍💻🚀